assets/airports/mdsd.json

```diff
@@ -0,0 +1,5080 @@
+            "suffix": {"17":"1J", "35":"1L"},
+            "rwy": {
+                "17": ["SD130", "SD131", ["SD135", "A40-"]],
+                "35": ["^SD140", "SD142", "SD143"]
+            },
+            "body": [],
+            "exitPoints": {
+                "KANAM": ["KANAM"]
+            },
+            "draw": [
+                ["SD130", "SD131", "SD135", "KANAM"],
+                ["SD140", "SD142", "SD143", "KANAM"]
+            ]
+        },
+        "KARIB1": {
+            "icao": "KARIB1",
+            "name": "Kareeb One",
+            "rwy": {
+                "35": []
+            },
+            "body": [["^PITOX", "A30-"], "CDO"],
+            "exitPoints": {
+                "KARIB": ["KARIB"]
+            },
+            "draw": [
+                ["CDO", "KARIB"]
+            ]
+        },
+        "KARIB2": {
+            "icao": "KARIB2",
+            "name": "Kareeb Two",
+            "rwy": {
+                "17": []
+            },
+            "body": [["^PODIS", "A30-"], "_CDO167015"],
+            "exitPoints": {
+                "KARIB": ["KARIB"]
+            },
+            "draw": []
+        },
+        "KERSO": {
+            "icao": "KERSO",
+            "name": "Kerso",
+            "suffix": {"17":"1J", "35":"1L"},
+            "rwy": {
+                "17": ["SD130", "SD131", ["SD132", "A40-"], ["SD133", "A60+"]],
+                "35": ["^SD140", ["SD141", "A35-"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "KERSO": ["KERSO"]
+            },
+            "draw": [
+                ["SD131", "SD132", "SD133", "KERSO"],
+                ["SD140", "SD141", "KERSO"]
+            ]
+        },
+        "KERSO1": {
+            "icao": "KERSO1",
+            "name": "Kerso One",
+            "rwy": {
+                "35": []
+            },
+            "body": [["^PITOX", "A30-"], "_CDO008015"],
+            "exitPoints": {
+                "KERSO": ["KERSO"]
+            },
+            "draw": [
+                ["CDO", "KERSO"]
+            ]
+        },
+        "KERSO2": {
+            "icao": "KERSO2",
+            "name": "Kerso Two",
+            "rwy": {
+                "17": []
+            },
+            "body": [["^PODIS", "A30-"], "CDO"],
+            "exitPoints": {
+                "KERSO": ["KERSO"]
+            },
+            "draw": []
+        },
+        "KODIX1": {
+            "icao": "KODIX1",
+            "name": "Kohdeex One",
+            "rwy": {
+                "35": []
+            },
+            "body": [["^PITOX", "A30-"], "_CDO338015"],
+            "exitPoints": {
+                "KODIX": ["KODIX"]
+            },
+            "draw": [
+                ["CDO", "PITOX", "KODIX"]
+            ]
+        },
+        "KODIX2": {
+            "icao": "KODIX2",
+            "name": "Kohdeex Two",
+            "rwy": {
+                "17": []
+            },
+            "body": [["^PODIS", "A30-"], "CDO", "PITOX"],
```

Comment:
I think here with the `KODIX2` we could go for the addition of `"_CDO335015": ["N18.64333333333", "W69.81066666667"]` to get them fully established on the airway as shown in the chart. Screenshot of the point we would be adding:
![image](https://user-images.githubusercontent.com/5103735/28037388-1b731e80-6589-11e7-9a49-36c65458bf96.png)